v2/pkg/additional/new.go

```diff
@@ -0,0 +1,23 @@
+package additional
+
+import (
+	"github.com/openshift/oc-mirror/v2/pkg/api/v1alpha2"
+	clog "github.com/openshift/oc-mirror/v2/pkg/log"
+	"github.com/openshift/oc-mirror/v2/pkg/mirror"
+	"github.com/openshift/oc-mirror/v2/pkg/manifest"
+)
+
+
+func New(log clog.PluggableLoggerInterface,
+		config v1alpha2.ImageSetConfiguration,
+		opts mirror.CopyOptions,
+		mirror mirror.MirrorInterface,
+		manifest manifest.ManifestInterface,
+		localStorageFQDN string,
+) CollectorInterface {
+	if localStorageFQDN != "" {
```

Comment:
@aguidirh - this is good, with simple logic we can plugin different implementations

Reply:
Thanks!